pkg/asset/installconfig/platform.go

```diff
@@ -199,7 +199,7 @@ func (a *Platform) libvirtPlatform() (*asset.State, error) {
 		},
 	}
 
-	prompt := asset.UserProvided{
+	uriPrompt := asset.UserProvided{
 		Question: &survey.Question{
 			Prompt: &survey.Input{
 				Message: "URI",
```

Comment:
The help makes this clear, but it might be worth updating this message to be `Libvirtd URI` to help folks who just need a small hint ;).

Reply:
personally i feel `URI` is enough. Do you have strong opinions on renaming it to `LibvirtdURI`

Reply:
I meant to rename this to Libvirt Connection URI a while back.